modules/nextflow/src/main/groovy/nextflow/trace/TraceObserver.groovy

```diff
@@ -123,11 +123,18 @@ trait TraceObserver {
     void onFlowError(TaskHandler handler, TraceRecord trace){}
 
     /**
-     * Method that is invoked when a value is published from a channel.
+     * Method that is invoked when a workflow output is published.
      *
+     * @param name
+<<<<<<< HEAD
+     *      The name of the workflow output
+=======
+>>>>>>> 3289e9e30 (first POC output structure and annotations)
```

Comment:
Oops, failed merge ☝️ (?)

Reply:
Fixed, I don't know why IntelliJ didn't warn me about it.